ext/mysqli/tests/mysqli_affected_rows.phpt

```diff
@@ -78,8 +78,7 @@ mysqli
     if (1 !== ($tmp = mysqli_affected_rows($link)))
         printf("[025] Expecting int/1, got %s/%s\n", gettype($tmp), $tmp);
 
-    $charsets = array('utf8');
-    foreach ($charsets as $k => $charset) {
+    foreach (['utf8'] as $charset) {
```

Comment:
Can't this foreach be eliminated now? Or do you want to do this as a follow-up?

Reply:
I didn't want to touch it further. If you'd like to find a way to remove you can do it as a follow-up. The `continue` statement deterred me from touching this piece of code.

Reply:
Makes sense, could use the typical C-ism of having a do-while loop with break statements, but lets leave this for a follow-up.